src/main/resources/templates/index.html

```diff
@@ -8,11 +8,11 @@
     <title>Inventory Manager</title>
 
 
-    <!--Dan's stuff-->
+    <!--Item form-->
```

Comment:
Made comments clearer so they are more useful to people viewing the code. This is a principle of clean code. Source: https://www.baeldung.com/java-clean-code